build.gradle

```diff
@@ -112,6 +112,7 @@ composeUp {
 	}
 
 	dependsOn checkForLiferayLicense
+	dependsOn importContainerData
```

Comment:
It seems strange, given that we're importing database dumps automatically, but we actually don't want to import the container data automatically.

Unlike the logic for importing database dumps, there's none for importing container data to check to see if the volume is already populated, so it will just override any work the CSE has already done in their bundle, forcing them to start over every time. Thus, we want the CSE to perform that work manually.